### PR TITLE
Enable WASM build and tests (OSSM-1931)

### PR DESCRIPTION
**Pull request by dcillera**
**Uncomment WASM build and tests**

**OSSM-1931**

Build and test are locally successful

### CODE REVIEW

File: test/envoye2e/stats_plugin/stats_test.go [hunk 1]
```diff
@@ -61,11 +61,11 @@ var Runtimes = []struct {
 	},
 	// https://issues.redhat.com/browse/MAISTRA-2140
 	// FIXME: Reenable when the issue above is fixed
-	// {
```
Comment (jwendell) [inline test/envoye2e/stats_plugin/stats_test.go:64]: These changes are not related, see the FIXME line above.

**Review by jwendell — CHANGES_REQUESTED** [review]
Did you start the local builder container with `--network=none`? We should make sure the bazel build/test commands do not pull anything from the internet.

File: test/envoye2e/stats_plugin/stats_test.go (hunk 1 above)
Comment (dcillera) [reply test/envoye2e/stats_plugin/stats_test.go:64]: No, the network was open

Comment (dcillera) [reply test/envoye2e/stats_plugin/stats_test.go:64]: Build to be restarted with that options and without modifications in stats_test.go

File: maistra/vendor/emsdk/emscripten_deps.bzl [hunk 2]
```diff
@@ -2,6 +2,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@build_bazel_rules_nodejs//:index.bzl", "npm_install", "node_repositories")
 load(":revisions.bzl", "EMSCRIPTEN_TAGS")
 
+#load("//bazel:repositories.bzl", )
+load("@envoy//bazel:repositories.bzl", "envoy_dependencies", "BUILD_ALL_CONTENT")
```
Comment (jwendell) [inline maistra/vendor/emsdk/emscripten_deps.bzl:5]: We should not touch files under the `vendor` directory. All these files come from Envoy and and changes made here will be overwritten when syncing with Envoy.
